cmd/ef_tests/test_runner.rs

```diff
@@ -164,31 +170,19 @@ fn check_poststate_against_db(test_key: &str, test: &TestUnit, db: &Store) {
             );
         }
     }
+    // Check lastblockhash is in store
+    let last_block_number = db.get_block_number(test.lastblockhash).unwrap();
```

Comment:
I would call `db.get_latest_block_number` and compare it with the lastblockhash

> `latest`: The most recent block in the canonical chain observed by the client, this block may be re-orged out of the canonical chain even under healthy/normal conditions;

We don't want to check that the lastblockhash is in store, instead we want 

> Compare the hash of the current head of the chain against [lastblockhash](https://github.com/lambdaclass/ethereum_rust/issues/232#-lastblockhash-hash), if they do not match, fail the test.

Reply:
Ok, makes sense